app/src/main/java/org/schabi/newpipe/ReCaptchaActivity.java

```diff
@@ -89,74 +90,89 @@ protected void onCreate(Bundle savedInstanceState) {
         myWebView.loadUrl(url);
     }
 
-    private class ReCaptchaWebViewClient extends WebViewClient {
-        private final Activity context;
-        private String mCookies;
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        boolean ret = super.onCreateOptionsMenu(menu);
+
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true);
+            actionBar.setHomeAsUpIndicator(getResources().getDrawable(R.drawable.ic_done_white_24dp));
```

Comment:
I was not able to get a reCAPTCHA while visiting peertube.co.uk, so I could not test your PR. However from what I can say when reading the code, it looks like there is no theme handling in the toolbar. Please ensure to have the correct button color when using the light theme.

Reply:
Unfortunately I am unable to test, too (I keep getting Network Errors in peertube.co.uk). But since the app bar background color is always red I don't think it is a problem to have an always white icon.

Reply:
Oh, apparently other menu icons are black in white theme even though they are on the exact same background color as dark theme, which is strange. With white theme this is, respectively, how the app and the recaptcha activity look: should I make the recaptcha activity's icon/text color black?

<table style="width:100%">
  <tr>
    <th><img src="https://user-images.githubusercontent.com/36421898/73567740-44835780-4467-11ea-9de9-b0b009c91d9d.png" width="200"/></th>
    <th><img src="https://user-images.githubusercontent.com/36421898/73567739-43522a80-4467-11ea-8dde-553408d72e5b.png" width="200"/></th>
  </tr>
</table>

Reply:
I agree that the black text looks not good. But I'd suggest to use the same colors as in the current theme for consistency. Could you open an issue to ask for help polishing the light theme?